about.html

```diff
@@ -0,0 +1,113 @@
+      <p class="send-email">Please contact us per Email for any further questions about MotoGP Grand Prix 2024!<br>
+        <a href="https://www.motogp.com/en/contact">MotoGP.contact</a></p>
+    </section>
+
+    <section class="event">
+      <div class="event-about">
+        <h2>
+          MotoGP <br> Monster Energy British Grand Prix.
+        </h2>
+        <hr>
+        <p class="motogp-history">The 1949 Grand Prix motorcycle racing season was the inaugural F.I.M. Road Racing World Championship Grand Prix season. The season consisted of six Grand Prix races in five classes: 500cc, 350cc, 250cc, 125cc and Sidecars 600cc. It began on 17 June, with Isle of Man TT and ended with Nations Grand Prix on 4 September.</p>
+      </div>
+      <div class="about-logo">
+        <img src="icons/MotoImg/Moto_Gp_logo.svg.png" alt="motoGP-logo">
+        <div class="about-logo-text">
+          <p>MotoGP</p>
+          <p>British Grand Prix 2023</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="past-races">
+      <h2>See some past races</h2>
+      <hr>
+      <p>Take a look at the past two MotoGP races and read the stories of our students</p>
+         <ul>
+          <li><a href="#">MotoGP Grand Prix<h3>2023</h3></a></li>
+          <li><a href="#">MotoGP Grand Prix<h3>2022</h3></a></li>
+          <li><a href="#">MotoGP Grand Prix<h3>2021</h3></a></li>
+        </ul>
+    </section>
+
+    <section id="sponsors" class="hidden">
+      <h2>Official Sponsors</h2>
+      <hr>
+      <ul class="sponsor-list">
+        <li><a href="https://www.tissotwatches.com/en-us/">Tissot</a></li>
+        <li><a href="https://www.michelinmotorsport.com/en/motorsport/">Michelin</a></li>
+        <li><a href="https://estrellagalicia00.es/">Estrella Galicia</a></li>
+        <li><a href="https://www.dhl.com/global-en/home.html">DHL</a></li>
+      </ul>
+    </section>
+  </main>
+  <footer class="about">
+    <img class="footer-logo" src="icons/MotoImg/Fédération_Internationale_de_Motocyclisme_logo.svg.png" alt="" class="footer-desktop-img">
+    <div>
+      <p>@2023 Dorna Sports SL. All rights reserved <br> All trademarks are the property of their respective owners.</p>
+      <p class="privacy-policy hidden">For details about how we use your information, please see our <a href="#">Privacy Policy</a></p>
+      <p class="cookies-terms hidden"><a href="#">Terms & Conditions</a> | <a href="#">Cookies</a></p>
+      </div>
+  </footer>
+<script src="./JS/index.js"></script>
+</body>
+</html>
```

Comment:
As suggested by the previous reviewer, please ensure to stick to the design as closely as possible .  kindly adjust the images section on the top of the images below to be on one row in the desktop view to match the given template. You currently have three images, meanwhile the images are supposed to be just two 👍 

Your design :
![image](https://github.com/JohnteFresh/The-Grand-Prix/assets/20537712/b5bd1ffd-0533-4145-a851-c63e88b0d01f)

Design given :

![Uploading image.png…]()